src/prelude.rs

```diff
@@ -14,7 +14,7 @@ pub use crate::{
 
 pub use teloxide_core::{
     adaptors::AutoSend,
-    requests::{Request, ResponseResult},
+    requests::Request,
```

Comment:
```suggestion
```

It's already in core's prelude: https://docs.rs/teloxide-core/0.2.1/teloxide_core/prelude/trait.Request.html